lib/resources.rb

```diff
@@ -108,13 +108,13 @@ def get_resource(key)
         end
       end
 
-      def authorization_types(practitioner: true, patient: false, system: false)
+      def authorization_types(provider: true, patient: false, system: false)
         html = '<div class="auth-types">'
         prev = false
 
-        if practitioner
-          html += '<a href="/authorization/#requesting-authorization-on-behalf-of-a-user" class="practitioner">'\
-                  'Practitioner</a>'
+        if provider
```

Comment:
while we're here, what do we think of beautifying this to the following:

```
      def authorization_types(provider: true, patient: false, system: false)
        template = '<a href="/authorization/#requesting-authorization-on-behalf-of-a-%s" class="%s">%s</a>'

        auth_types = []

        auth_types << template % %w[user provider Provider] if provider
        auth_types << template % %w[user patient Patient] if patient
        auth_types << template % %w[system system System] if system

        "<div class=\"auth-types\">#{auth_types.join(' | ')}</div>"
      end
```

Reply:
https://github.com/cerner/fhir.cerner.com/pull/430/commits/56870f0ff4573ad640d54226948788ec51129ab0#diff-7718c11ed130ac00a9854e0b089f74b4R112

Reply:
fixed lint a040879